attendance/src/main/java/org/robojackets/apiary/attendance/model/AttendanceViewModel.kt

```diff
@@ -118,15 +120,20 @@ class AttendanceViewModel @Inject constructor(
         }
     }
 
-    fun loadAttendables(attendableType: AttendableType) {
+    fun loadAttendables(attendableType: AttendableType, forceRefresh: Boolean = false) {
         error.value = null
         loadingAttendables.value = true
         viewModelScope.launch {
-            if (attendableType == AttendableType.Team && attendableTeams.value.isNullOrEmpty()) {
+            if (attendableType == AttendableType.Team &&
+                (attendableTeams.value.isEmpty() || forceRefresh)) {
                 meetingsRepository.getTeams().onSuccess {
                     attendableTeams.value = this.data.teams
                         .filter { it.attendable }
                         .sortedBy { it.name }
+
+                    missingHiddenTeams.value = attendableTeams.value.none {
+                        it.name.contains("training", ignoreCase = true)
+                    }
```

Comment:
What is this doing?

Reply:
If true, the "some teams missing" callout appears on the attendable selection screen

Reply:
Can you do a permissions check instead?

Reply:
It would be significant effort and add latency until there's an API caching layer in the app. Given that our teams list rarely changes, it is much easier to use training teams as a bellwether instead

Reply:
Hmm. It looks like there's a `visible` attribute returned from the API...can you use that?

Reply:
I could rename it to be more clear

Reply:
I mean you could check for the `visible` flag being false on a team object

Reply:
I can do that